async_processor/worker.py

```diff
@@ -162,7 +162,7 @@ async def _process_single_step(self, input_: Input, output: Optional[Output]):
                 )
 
     async def run(self, stop_event: asyncio.Event):
-        logger.info("Polling messages")
+        logger.debug("Polling messages")
```

Comment:
Minor, start and end messages that print only once can probably stay info. We can improve the logging formatted to include `[sidecar]`